[2024-05-01T23:32:04.216-0500] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-01T23:32:04.223-0500] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 scheduled__2024-05-01T00:00:00+00:00 [queued]>
[2024-05-01T23:32:04.226-0500] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 scheduled__2024-05-01T00:00:00+00:00 [queued]>
[2024-05-01T23:32:04.226-0500] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-01T23:32:04.231-0500] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): runme_2> on 2024-05-01 00:00:00+00:00
[2024-05-01T23:32:04.234-0500] {standard_task_runner.py:63} INFO - Started process 34816 to run task
[2024-05-01T23:32:04.237-0500] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_2', 'scheduled__2024-05-01T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', '/Users/nestoralfaro/Desktop/pbi-scenario-two/py_env/lib/python3.9/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/var/folders/14/31_1v10s5nvcqyswsggcn5vh0000gn/T/tmp5llj36h4']
[2024-05-01T23:32:04.237-0500] {standard_task_runner.py:91} INFO - Job 4: Subtask runme_2
[2024-05-01T23:32:04.255-0500] {task_command.py:426} INFO - Running <TaskInstance: example_bash_operator.runme_2 scheduled__2024-05-01T00:00:00+00:00 [running]> on host nestors-macbook.attlocal.net
[2024-05-01T23:32:04.280-0500] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='runme_2' AIRFLOW_CTX_EXECUTION_DATE='2024-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-01T00:00:00+00:00'
[2024-05-01T23:32:04.280-0500] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-01T23:32:04.281-0500] {subprocess.py:63} INFO - Tmp dir root location: /var/folders/14/31_1v10s5nvcqyswsggcn5vh0000gn/T
[2024-05-01T23:32:04.281-0500] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "example_bash_operator__runme_2__20240501" && sleep 1']
[2024-05-01T23:32:04.284-0500] {subprocess.py:86} INFO - Output:
[2024-05-01T23:32:04.286-0500] {subprocess.py:93} INFO - example_bash_operator__runme_2__20240501
[2024-05-01T23:32:05.296-0500] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-01T23:32:05.302-0500] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-01T23:32:05.345-0500] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20240501T000000, start_date=20240502T043204, end_date=20240502T043205
[2024-05-01T23:32:05.372-0500] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-01T23:32:05.393-0500] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-01T23:32:05.394-0500] {local_task_job_runner.py:222} INFO - ::endgroup::
